### DISCUSSION TITLE
Google Play Services / Capacitor

### DISCUSSION THREAD

**Discussion by Ste-denko** [HaylLtd/cordova-plugin-background-geolocation#54] [Q&A]
Ahoi,

are google play services required for the usage with Android?
And anyone maybe positive or negative experience with Capacitor usage of this plugin?

Thanks and cheers
St.

**Comment by Ste-denko** [comment]
Anyone an info about that?
I would really appreciate it ...
Thanks again.

**Reply by HarelM** [reply]
I think some google services are needed for fused location, not sure though.
I don't have experience with capacitor, this is a standard cordova plugin so in theory it should work but I have seen reports of this not working.
Any help would be appreciated regarding capacitor.
